backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxInitializerApi.scala

```diff
@@ -40,6 +40,10 @@ class VeloxInitializerApi extends InitializerApi {
         new VeloxSharedlibraryLoaderCentos7
       } else if (system.contains("alinux") && system.contains("3")) {
         new VeloxSharedlibraryLoaderCentos8
+      } else if (system.contains("Anolis") && system.contains("8")) {
+        new VeloxSharedlibraryLoaderCentos8
+      } else if (system.contains("Anolis") && system.contains("7")) {
```

Comment:
Alinux3 is verified with Alibaba cloud EMR release version [5.11.1](https://developer.aliyun.com/article/1182969?spm=5176.21211227.J_6302206100.1.7a6c123ciSdv41)
I'm not sure Anolis7/8 is supported too.

How do you test it?

Reply:
@Yohahaha Thanks for your review, we use our own basic image with anolis 8 with spark and only modify the VeloxInitializerApi in gluten and do not modify centos set.up script.

Reply:
Thanks for clarify, make sense for me.
cc @FelixYBW @zhejiangxiaomai